bld/CLMBuildNamelist.pm

```diff
@@ -2397,6 +2425,12 @@ sub setup_logic_do_transient_crops {
       $log->fatal_error($cannot_be_true);
    }
 
+   if (&value_is_true($nl->get_value($var))) {
+      if ($n_dom_pfts > 0 || $n_dom_landunits > 0 || $toosmall_soil > 0 || $toosmall_crop > 0 || $toosmall_glacier > 0 || $toosmall_lake > 0 || $toosmall_wetland > 0 || $toosmall_urb_tbd > 0 || $toosmall_urb_hd > 0 || $toosmall_urb_md > 0) {
```

Comment:
What is trying to limited here? I'm pretty sure having n_dom_pfts and n_dom_landunits together is fine. And I think any of the toosmall's with n_dom_pfts should be fine (other than maybe toosmall_soil). I see the point of checking the toosmalls with n_dom_landunits. I'm not sure if there are problems with the other conditions though. So we should clarify what we are trying to check here.

Reply:
Added comments:
`if do_transient_pfts is .true. and any of these (n_dom_* or toosmall_*) are > 0, then give fatal error`
`if do_transient_crops is .true. and any of these (n_dom_* or toosmall_*) are > 0, then give fatal error`

Reply:
I see how n_dom_landunits, could interfere with the too-smalls, but I don't see how n_dom_pfts could interfere. Is there really a problem with setting both n_dom_pfts and the too-smalls?

Reply:
Pls note that these comments say nothing of the n_doms interfering with the toosmalls. Instead they address the incompatibility with transient: The n_doms and the toosmalls are incompatible with **transient**.

@ekluzek pls explain what you mean by "n_dom_landunits could interfere with the toosmalls."

This was my response to the same concern elsewhere in the PR:
"I disagree that the toosmalls may not be used with n_dom_pfts and n_dom_landunits. Here's an example: A user may wish to run with 2 dominant pfts and 2 dominant landunits and may still want to specify that a particular landunit shall be used if greater than 10%. As written, the code can do that."

Reply:
@ekluzek let me know if it would be helpful to go over some of this in person by phone. If so, feel free to book a time on my google calendar.